### ISSUE TITLE
Idempotent version of "last n versions"

### ISSUE THREAD

**Issue by AprilArcus** [browserslist/browserslist#680]
"Last _n_ versions" refers to the last _n_ versions as of a "now" which corresponds to the moment the build is initiated. However, once my team cuts a release branch, it is necessary for our set of supported browsers to be frozen at this point. It would simplify matters for us to be able to specify an absolute date which would serve as the "now" against which "last _n_ versions" queries can be evaluated.

**Comment by ai** [comment]
If you will not change `caniuse-lite` version (for instance, by locking version in npm lock file) the queries result will not be changed.

Does it works for you?

**Comment by AprilArcus** [comment]
Yes, although it is irritating and potentially misleading for other devs when old versions trigger the warning emitted at https://github.com/browserslist/browserslist/blob/4.20.3/node.js#L381-L386

**Comment by ai** [comment]
We believe that updating browsers twice per year will reduce polyfills and is good for website performance.

You can lock browsers between this updates.

Why do you need to keep browsers locked for now than 6 month?

**Comment by AprilArcus** [comment]
We have LTS releases that need to receive security backports for up to a year. The set of browsers that was supported when the LTS was initially released should not change as a result of backporting a security fix, so we want to freeze `caniuse-lite` in place for the release's entire lifetime.

**Comment by ai** [comment]
> The set of browsers that was supported when the LTS was initially released should not change as a result of backporting a security fix

What do you think about locked browser versions in Browserslist config instead?

```sh
npx browserslist "defaults" > .browserslistrc
```

You will have the config like:

```
and_chr 102
and_ff 101
and_uc 12.12
android 101
chrome 102
chrome 101
chrome 100
edge 102
edge 101
firefox 101
firefox 100
firefox 91
ios_saf 15.5
ios_saf 15.4
```

It will be explicit that you are locked with exact versions.
